### PR TITLE
[Maps] Add locationV2 to DiscoverySidebar + tweak styles

### PR DESCRIPTION
**Pull request by jshoe**
### Description
- Add locationV2 to the right summary info DiscoverySidebar.
- Add a message to MapPreviewSidebar when there's no data.
- Update color of the bubble icon and style of the map toggler according to the mocks.
- Tweak name truncation method.

### Demo
Screen Shot 2019-06-06 at 5 08 08 PM
Screen Shot 2019-06-06 at 6 19 53 PM

### Test and Reproduce
- Go to data discovery samples tab, find locationV2 in the right pane sidebar. Go to the map and observe the new styles.

### CODE REVIEW

File: app/assets/src/components/views/discovery/mapping/MapPreviewSidebar.jsx [hunk 1]
```diff
@@ -218,10 +218,20 @@ export default class MapPreviewSidebar extends React.Component {
     );
   };
 
+  renderNoData = () => {
+    return (
+      <div className={cs.noData}>
+        Select a location to see summary info and samples.
```
Comment (tfrcarvalho) [inline app/assets/src/components/views/discovery/mapping/MapPreviewSidebar.jsx]: When does this happen (samples.length = 0)? 
Shouldn't by default just see all samples?

File: app/assets/src/components/views/samples/SamplesView.jsx [hunk 2]
```diff
@@ -385,12 +386,9 @@ class SamplesView extends React.Component {
...
               )}
             >
-              <i
-                className={cx(
-                  "fa",
-                  display === "map" ? "fa-globe" : "fa-list-ul",
-                  cs.icon
-                )}
+              <Icon
+                name={display === "map" ? "fa-globe" : "fa-list-ul"}
```
Comment (tfrcarvalho) [inline app/assets/src/components/views/samples/SamplesView.jsx]: Can you define an icon for this?

File: app/assets/src/components/views/discovery/mapping/MapPreviewSidebar.jsx (hunk 1 above)
Comment (jshoe) [reply app/assets/src/components/views/discovery/mapping/MapPreviewSidebar.jsx]: oh yeah I broke that off as a separate task (pulling in what's already in DiscoveryView) and I think I'll add that soon in a separate PR

File: app/assets/src/components/views/samples/SamplesView.jsx (hunk 2 above)
Comment (jshoe) [reply app/assets/src/components/views/samples/SamplesView.jsx]: 👍

**Pull request merged**